src/runtime/link-accounts-flow.js

```diff
@@ -206,13 +206,20 @@ export class LinkSaveFlow {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
-    /** @private {!../api/subscriptions.SaveSubscriptionRequest} */
-    this.request_ = request;
+    /** @private {!../api/subscriptions.SaveSubscriptionRequestCallback} */
+    this.callback_ = callback;
+
+    /** @private {?Promise<!../api/subscriptions.SaveSubscriptionRequest>} */
+    this.requestPromise_ = null;
 
     /** @private {?ActivityIframeView} */
     this.activityIframeView_ = null;
   }
 
+  /** @return {?Promise<!../api/subscriptions.SaveSubscriptionRequest>} */
+  getRequestPromise() {
```

Comment:
Is it necessary to export it as a public API? Unfortunately currently all public APIs are preserved in the minified version, but it looks like this method is only needed for testing?

Reply:
done